tensorflow/compiler/xla/service/gpu/launch_dimensions.h

```diff
@@ -61,22 +57,23 @@ class LaunchDimensions {
            thread_counts_per_block_.z;
   }
 
-  bool last_dim_aligned_for_vectorization() const {
-    return last_dim_aligned_for_vectorization_;
-  }
-
  private:
   Dim3D block_counts_;
   Dim3D thread_counts_per_block_;
-  bool last_dim_aligned_for_vectorization_;
 };
 
 std::ostream& operator<<(std::ostream& out,
                          const LaunchDimensions& launch_dims);
 
+struct LaunchDimensionsConfig {
+  int unroll_factor = 1;
+  bool few_waves = false;
```

Comment:
Is it possible to move the documentation string to here as to what `few_waves` means?